### PR TITLE
Create contributor_fetch.yml

### PR DESCRIPTION
**Pull request by Darshan-upadhyay1110**
1. **Fetch Contributors**: The script retrieves the list of contributors for the `CollaboraOnline/online` repository using GitHub's API.

2. **Generate Markdown File**: It creates a Markdown file, `contributors.md`, which includes the names, contributions, and GitHub avatars of the contributors.

3. **Automated Workflow**: The script is set up to run automatically as part of a scheduled workflow in GitHub Actions, updating the contributors credits page regularly. 

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] I have run `make prettier-write` and formatted the code.
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by Darshan-upadhyay1110** [comment]
Tested in my fork repo it's working 🎉 
https://github.com/Darshan-upadhyay1110/online/actions/runs/11038323869/job/30661245721

File: .github/workflows/contributor_fetch.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        run: |
+          cp contributors.md repo/content/post/contributors.md  # Copy to the desired path
+
+      - name: Commit changes
+        run: |
+          cd repo
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+          git add content/post/contributors.md
+          git commit -m "Update contributors list [skip ci]"
```
Comment (eszkadev) [inline .github/workflows/contributor_fetch.yml:53]: do we need sign off too ? `-s`

Comment (Darshan-upadhyay1110) [reply .github/workflows/contributor_fetch.yml:53]: @eszkadev i don't know if we need to do sign-off or not ?  But i see IO does not have any CI check for Sign off commit....

It's good to have it sign off let me add it

**Comment by pedropintosilva** [comment]
This doesn't affect the Collabora Online. It just about the credits we can merge even if it has problems

**Review by pedropintosilva — APPROVED** [review]
thanks!

**Pull request merged**